src/includes/performance/distributed-tracing/java.mdx

```diff
@@ -0,0 +1,23 @@
+## Distributed Tracing
+
+Traces can bridge across multiple software services. Each span in a trace can be represented as a `sentry-trace` header, containing the trace id, span id, and sampling details. This `sentry-trace` header can be passed to downstream services so that they can create spans that are a continuation of the trace started in the originating service.
+
+To obtain a trace header from the span, use `ISpan#toTraceHeaders()` method. Then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` (also available as a constant `SentryTraceHeader.SENTRY_TRACE_HEADER`) HTTP header.
+
+To create a span as a continuation of the trace retrieved from the upstream service, pass the `sentry-trace` header value to the transaction context:
+
+```java
+String sentryTrace = request.getHeader(SentryTraceHeader.SENTRY_TRACE_HEADER);
```

Comment:
@maciejwalkowiak this wont work ootb because `new SentryTraceHeader` throws `InvalidSentryTraceHeaderException` on its signature, we need to add a try/catch in the example